sqlx-sqlite/src/connection/establish.rs

```diff
@@ -296,6 +296,8 @@ impl EstablishParams {
             log_settings: self.log_settings.clone(),
             progress_handler_callback: None,
             update_hook_callback: None,
+            #[cfg(feature = "preupdate-hook")]
+            preupdate_hook_callback: None,
```

Comment:
I don't see this being set or referenced by anything. Did you mean to expose this on `SqliteConnectOptions`?

Reply:
This was being used within the `preupdate_hook` module to avoid having to add more `cfg` checks in the main connection module and make additional fields `pub(super)`, but that does make it a bit harder to find. I went ahead and moved that logic to be with the rest of the hooks.